dask/bytes/tests/test_local.py

```diff
@@ -70,6 +70,17 @@ def test_with_urls():
         assert sum(map(len, values)) == len(files)
 
 
+def test_with_paths():
+    pathlib = pytest.importorskip('pathlib')
+    with filetexts(files, mode='b'):
+        url = pathlib.Path('./.test.accounts.*')
+        sample, values = read_bytes(url, blocksize=None)
+        assert sum(map(len, values)) == len(files)
+    with pytest.raises(OSError):
```

Comment:
So is it possible, when urlsplit returns, to raise an error if the scheme is non-empty and the original object was a Path? Or would that break the internal abstractions?